modules/forms/src/models/forms.ts

```diff
@@ -140,6 +140,15 @@ export interface CheckboxField extends FormRequiredFieldModel {
     [extra: string]: any;
 }
 
+/**
+ * Query Parameters field model
+ */
+export interface QueryParamsField extends FormRequiredFieldModel {
```

Comment:
```suggestion
export interface QueryParamsFieldInterface extends FormRequiredFieldModel {
```

Reply:
I used this because it is the same pattern followed in the module.